openfecwebapp/templates/legal-advisory-opinion.html

```diff
@@ -62,6 +54,37 @@ <h2 class="t-ruled--bold t-ruled--bottom">Documents</h2>
       <h2 class="t-ruled--bold t-ruled--bottom">Entities</h2>
       Coming soon.
     </section>
+    <section class="content__section">
+      <aside id="legal-citations" class="sidebar sidebar--primary">
+          <h4 class="sidebar__title">Legal citations</h4>
+          <div class="sidebar__content">
+            <div class="grid grid--2-wide">
+                <div class="grid__item">
+                  <p class="t-bold">This opinion cites earlier opinions</p>
+                  <div class="rich-text">
+                    {% if advisory_opinion.citations %}
+                    {% for citation in advisory_opinion.citations %}
+                      <p><a href="/legal/advisory-opinions/{{ citation.no }}">AO {{ citation.no }}</a><br><i>{{ citation.name }}</i></p>
+                    {% endfor %}
+                    {% else %}
+                      <p>Does not cite other opinions</p>
+                    {% endif %}
+                  </div>
+                </div>
+                <div class="grid__item">
+                  <p class="t-bold">This opinion is cited by these later opinions</p>
```

Comment:
No, that is the title of the section, not an error message: <img width="989" alt="screen shot 2017-01-24 at 3 05 31 pm" src="https://cloud.githubusercontent.com/assets/2559082/22266719/a7aa3f6c-e246-11e6-880d-c73c7d80fcaf.png">